src/NodaTime.Demo/PeriodDemo.cs

```diff
@@ -177,5 +177,28 @@ public void Between_LocalDateTimesWithGivenUnits()
             Assert.AreEqual(23, period.Hours);
             Assert.AreEqual("P2Y264DT23H", period.ToString());
         }
+
+        [Test]
+        public void Normalize()
+        {
+            var original = new PeriodBuilder { Weeks = 2, Days = 5 }.Build();
+            var expected = new PeriodBuilder { Days = 19 }.Build();
```

Comment:
Just thinking about this and your other snippets for `DateAdjuster`, we may well not see any use of `expected` given the conversion from `Assert.AreEqual` to `Console.WriteLine`. I'll hold off merging this until we've seen what those snippets look like - it may well be worth inlining the expected value into `Assert.AreEqual`.

Reply:
Proved to be correct decision. I have inlined them now.